apps/api/src/database/factories/user.ts

```diff
@@ -1,14 +1,23 @@
-import { PrismaClient, Subscription, User } from '@prisma/client'
+import { Subscription, User } from '@prisma/client'
 import { Factory } from 'fishery'
 import { faker } from '@faker-js/faker'
 
+import prisma from '../client'
```

Comment:
Does this change make us skip creating a new DB connections and instead re-use the same one?

Reply:
Yes, in the Backend API it already happens: because of NestJS DI capabilities it will inject a singleton of the prisma client. 

This is for the Model Factories that runs on db seeding, i.e. out of the scope of the framework.